ddmrp/models/stock_warehouse_orderpoint.py

```diff
@@ -226,6 +229,90 @@ def _compute_ddmrp_chart(self):
             script, div = components(p)
             rec.ddmrp_chart = '%s%s' % (div, script)
 
+    def _compute_ddmrp_demand_supply_chart(self):
+        for rec in self:
+            if not rec.buffer_profile_id:
+                # Not a buffer, skip.
+                rec.ddmrp_demand_chart = ''
+                rec.ddmrp_supply_chart = ''
+                continue
+
+            # Prepare data:
+            demand_data = rec._get_demand_by_days()
+            supply_data = rec._get_incoming_by_days()
+            width = timedelta(days=0.4)
+
+            # Plot demand data:
+            if demand_data:
+                x_demand = list(demand_data.keys())
+                y_demand = list(demand_data.values())
+
+                p = figure(plot_width=500, plot_height=400,
+                           y_axis_label='Quantity', x_axis_type='datetime')
+                p.toolbar.logo = None
+                p.sizing_mode = 'stretch_both'
+                # TODO: # p.xaxis.label_text_font = 'helvetica'
+                p.xaxis.formatter = DatetimeTickFormatter(
+                    hours=["%d %B %Y"], days=["%d %B %Y"], months=["%d %B %Y"],
```

Comment:
You can do same that here:
https://github.com/OCA/ddmrp/pull/39

Reply:
Will do now. Thanks